### ISSUE TITLE
Remove deprecate `twisted.trial.util.findObject`.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6260]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)| @tomprince reported|
|-|-|
|Trac ID|trac#6260|
|Type|task|
|Created|2013-01-18 03:29:30Z|
|Branch|https://github.com/twisted/twisted/tree/remove-findObject-6260|

It was deprecated in version 10.1.0, so lets get rid of it.

<details><summary>Searchable metadata</summary>

```
trac-id__6260 6260
type__task task
reporter__tom_prince tom.prince
priority__normal normal
milestone__ 
branch__branches_remove_findObject_6260 branches/remove-findObject-6260
branch_author__tomprince tomprince
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1358479770000000 1358479770000000
changetime__1358829223000000 1358829223000000
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

(In [36846]) Branching to remove-findObject-6260

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

> It was deprecated in version 10.1.0, so lets get rid of it.

As discussed on IRC, a better reason to get rid of it is that its unit tests are messy and dump warnings into the test suite output, which slightly burdens developers trying to interpret the output.  The fact that it was deprecated in 10.1.0 just means it is now _okay_ to remove it, not that we have to.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @tomprince<br>@exarkun set status to `new`|
|-|-|

Branch looks fine.  Please merge once a [build](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-findObject-6260) completes successfully.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_5"></a>|@tomprince set status to `closed`|
|-|-|

(In [36878]) Merge remove-findObject-6260: Remove deprecated `twisted.trial.util.findObject`.

Author: tom.prince
Reviewers: glyph
Fixes: [#6260](https://github.com/twisted/twisted/issues/6260)

Its unit tests are messy and dump warnings into the test suite output,
which slightly burdens developers trying to interpret the output.  It
was deprecated in version 10.1.0, so lets get rid of it.
